recipe/meta.yaml

```diff
@@ -1,37 +1,40 @@
-{% set version = "13.0.1" %}
+{% set version = "14.0.0" %}
 {% set major_ver = version.split(".")[0] %}
 
 package:
   name: llvm-package
   version: {{ version }}
 
 source:
-  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-{{ version.replace(".rc", "rc") }}.src.tar.xz
-  sha256: ec6b80d82c384acad2dc192903a6cf2cdbaffb889b84bfb98da9d71e630fc834
-  patches:
-    # - patches/intel-D47188-svml-VF.patch    # Fixes vectorizer and extends SVML support
-    # - patches/expect-fastmath-entrypoints-in-add-TLI-mappings.ll.patch # adjusts test added in 10.0.0 for intel-D47188-svml-VF.patch effects
-    # - patches/amd-roc-2.7.0.diff
-    - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
-    # Don't link libLLVM.dylib to llvm-config. Makes it easier for cross compiling
-    - patches/0002-Don-t-link-libLLVM.dylib-to-llvm-config.patch
+  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-{{ version.replace(".rc", "rc") }}.src.tar.xz
+    sha256: 4df7ed50b8b7017b90dc22202f6b59e9006a29a9568238c6af28df9c049c7b9b
+    patches:
+      # - patches/intel-D47188-svml-VF.patch    # Fixes vectorizer and extends SVML support
+      # - patches/expect-fastmath-entrypoints-in-add-TLI-mappings.ll.patch # adjusts test added in 10.0.0 for intel-D47188-svml-VF.patch effects
+      # - patches/amd-roc-2.7.0.diff
+      - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
+      # Don't link libLLVM.dylib to llvm-config. Makes it easier for cross compiling
+      - patches/0002-Don-t-link-libLLVM.dylib-to-llvm-config.patch
+  # Workaround for https://github.com/llvm/llvm-project/issues/53281
+  - url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-{{ version.replace(".rc", "-rc") }}.tar.gz
+    sha256: 35ce9edbc8f774fe07c8f4acdf89ec8ac695c8016c165dd86b8d10e7cba07e23
+    folder: llvm-project
 
 build:
-  number: 2
+  number: 0
   merge_build_host: false
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake
-    - ninja     # [win]
+    - ninja
```

Comment:
Avoid cycle. `libcxx -> llvm -> ninja -> libcxx`

Reply:
Ah, that's a pity. But why would it be a cycle in the first place? Ninja (currently) depends only on `libcxx >=11.1.0`, so it's not like that would be incompatible?

Reply:
(I mean that the versions don't need to match, so it's never a closed cycle AFAICT.)

Reply:
True